### PR TITLE
Redirect /opportunity pages to /initiatives

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #2971 

### English
https://foundation-s-issue-2971-2zw1sx.herokuapp.com/en/initiatives/page-has-been-moved-initiatives/ (link should redirect to /en/initiatives/...)
### French
https://foundation-s-issue-2971-2zw1sx.herokuapp.com/fr/initiatives/page-has-been-moved-initiatives/ (link should redirect to /fr/initiatives/...)

### English
https://foundation-s-issue-2971-2zw1sx.herokuapp.com/en/opportunity/page-still-under-opportunity/ (link redirects to /en/initiatives/... but it's a 404 since we haven't moved the page yet)
### French
https://foundation-s-issue-2971-2zw1sx.herokuapp.com/fr/opportunity/page-still-under-opportunity/  (link redirects to /fr/initiatives/... but it's a 404 since we haven't moved the page yet)

### CODE REVIEW

File: network-api/networkapi/wagtailpages/views.py [hunk 1]
```diff
@@ -22,3 +24,11 @@ def custom404_view(request, exception):
...
         html = render(request, '404.html')
         return HttpResponseNotFound(html.content)
+
+
+def redirect_to_initiatives(request, rest):
+    lang = request.LANGUAGE_CODE
+    translation.activate(lang)
+    request.session[translation.LANGUAGE_SESSION_KEY] = lang
+
+    return redirect(urljoin('/', 'initiatives/' + rest))
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/views.py]: do we need `urljoin` here? Can we use `f'/initiates/{rest}'`?

Comment (mmmavis) [reply network-api/networkapi/wagtailpages/views.py]: `f'/initiatives/{rest}'` works. PR has been updated.

**Comment by Pomax** [comment]
It's also doing pretty weird things when comparing `localhost:8000/opportunity/...` vs. `localhost:8000/en/opportunity/...` =(

**Comment by mmmavis** [comment]
Ummm both `test.example.com:8000/opportunity` and `test.example.com:8000/en/opportunity` redirected me to `test.example.com:8000/initiatives`. 🤔

**Comment by mmmavis** [comment]
PR updated!

@Pomax I can't reproduce the issues you were having. Can you try out this PR again and let me know if you still run into issues? 

@gideonthomas 

> We'll have to test on prod for edge cases.

What kind of edge cases?

> We have to manually move opportunity pages to initiatives right?

Yea, we'll have to manually move those pages using the "move" button on Wagtail admin first before landing this PR.

**Comment by gideonthomas** [comment]
edge cases that we don't know about haha. Basically I mean, run on prod and monitor if anything breaks. I don't see any issues with the implementation so the only thing I can think of that will break this is some random edge case we haven't thought about.

**Comment by Pomax** [comment]
Looks like one thing we're still missing is argument capture. I.e. this:

https://foundation-s-issue-2971-u9kqiv.herokuapp.com/pt/opportunity/test/tewo?filter=ckae

redirects to

https://foundation-s-issue-2971-u9kqiv.herokuapp.com/pt/initiatives/test/tewo

instead of 

https://foundation-s-issue-2971-u9kqiv.herokuapp.com/pt/initiatives/test/tewo?filter=ckae

File: network-api/networkapi/wagtailpages/views.py [hunk 2]
```diff
@@ -22,3 +23,15 @@ def custom404_view(request, exception):
...
+
+
+def redirect_to_initiatives(request, subpath):
+    lang = request.LANGUAGE_CODE
+    translation.activate(lang)
+    request.session[translation.LANGUAGE_SESSION_KEY] = lang
+    query_string = ""
+
+    if request.META['QUERY_STRING']:
+        query_string = "?%s" % request.META['QUERY_STRING']
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/views.py]: This is a legacy string templating solution from Python 2.7 and is highly discouraged for 3.x

Instead, f-strings are the recommended way to composite strings now:

```suggestion
        query_string = f'?{request.META['QUERY_STRING']}'
```
